_docs-v4/custom-views/visibleRange.md

```diff
@@ -28,9 +28,13 @@ You can also specify a function that dynamically generates a range from the curr
 var calendar = new Calendar(calendarEl, {
   defaultView: 'timeGrid',
   visibleRange: function(currentDate) {
+    // Generate a new date for manipulating, based on the current date
+    var startDate = new Date(currentDate.valueOf());
+    var endDate = new Date(currentDate.valueOf());
+
     return {
-      start: currentDate.clone().subtract(1, 'days'),
-      end: currentDate.clone().add(3, 'days') // exclusive end, so 3
+      start: startDate.setDate(startDate.getDate() - 1), // One day in the past
```

Comment:
this might give the impression that `setDate` is chainable, whereas it returns a ms time. i'll recommend another way to write this in the main comments..